File: practice-challenges/more-evens-or-odds/more_evens_or_odds.py
"""Solution for More Evens or Odds?."""


def more_evens_or_odds(nums):
    count_evens = 0
    count_odds = 0

    for num in nums:
        if num % 2 == 0:
            count_evens = count_evens + 1
        else:
            count_odds = count_odds + 1

    if count_evens > count_odds:
        return 'even'
    elif count_odds > count_evens:
        return 'odd'
    else:
        return 'even and odd'

File: practice-challenges/more-evens-or-odds/more-evens-or-odds.js
function moreEvensOrOdds(arr) {
  let evenCount = 0;
  let oddCount = 0;

  for (let i = 0; i < arr.length; i++) {
    if (arr[i] % 2 === 0) {
      evenCount += 1;
    } else {
      oddCount += 1;
    }
  }

  if (evenCount > oddCount) {
    return 'even';
  } else if (oddCount > evenCount) {
    return 'odd';
  } else {
    return 'even and odd'
  }
}